apps/dashboard/app/models/project.rb

```diff
@@ -25,6 +25,13 @@ def self.from_directory(dir)
     nil
   end
 
+  # TODO: Use it to populate similar page as /projects where we will keep the imported projects
+  def self.possible_imports
+    Rails.cache.fetch('possible_imports', expires_in: 1.hour) do
+      importable_directories
+    end
+  end
+
   class << self
```

Comment:
Instead of defining the function as `self.possible_imports` we should just move it down a few lines under the `class << self` directive. 

These 2 function definitions have the same affect of creating class methods.

```ruby
class << self
  def foo
    # ...
  end
end

def self.foo
  # ..
end
```